apm-sniffer/apm-sdk-plugin/resin-3.x-4.x-plugin/src/main/java/org/skywalking/apm/plugin/resin/v3x/v4x/ResinInterceptor.java

```diff
@@ -0,0 +1,65 @@
+package org.skywalking.apm.plugin.resin.v3x.v4x;
```

Comment:
The package name didn't fit the Java Package Tradition. In Java, `*.v3x.v4x` means **3** is major version and **4** is minor version. But, according from my understanding, **3** and **4** are all major versions.

So I prefer the package name is `org.skywalking.apm.plugin.resin34x`.  `TomcatInterceptor` class under package `org.skywalking.apm.plugin.tomcat78x` is for your reference.